nossas/bonde
# Feature purge xlsx package

# Related issues
- Remove xlsx #505

# How to test
- Open a mobilization that contains a form entry widget to edit in admin
- Click to edit the form entry widget
- Go to the report tab
- Click on the download button and, it should download the report file in `.csv` format
